server/src/main/java/io/crate/metadata/Schemas.java

```diff
@@ -327,6 +328,14 @@ public <T extends TableInfo> T getTableInfo(RelationName ident) {
         }
     }
 
+    public TranslogIndexer getTranslogIndexer(RelationName ident) {
+        TableInfo ti = getTableInfo(ident);
+        if (ti instanceof DocTableInfo tdi) {
+            return tdi.getTranslogIndexer();
+        }
+        return null;
```

Comment:
```suggestion
        DocTableInfo ti = getTableInfo(ident);
        return ti.getTranslogIndexer();
```


See https://github.com/crate/crate/blob/a1647ab7b8d4cc59407e631c9357c58dec266fd7/server/src/main/java/io/crate/metadata/Schemas.java#L321-L327

If `getTranslogIndexer` would be used with a system table, view or foreign table it would result in a proper error.